examples/produce_sketch_basic/README.md

```
@@ -32,6 +32,18 @@ pc inspect -s rect_01
 <td valign=top>The longest way to create a basic circle in PartCAD</td>
 </tr></table>
 
+### circle_04
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)